source/content.css

```diff
@@ -60,3 +60,8 @@ Dashboard
 	float: unset !important;
 	margin: 0 auto !important;
 }
+
+/* Custom Inline Image Element */
+.AdaptiveMedia .refined-instagram-inline {
```

Comment:
`.refined-instagram-inline` => `.refined-twitter_instagram-inline` To make it clear the class was added by the extension.